### ISSUE TITLE
MPII training from scratch gives no prediction 

### ISSUE THREAD

**Issue by Aakriti05** [ZheC/Realtime_Multi-Person_Pose_Estimation#222]
Hello,
I trained my model with 2500 images (120000 iteration), but the trained model is **not** giving any predictions on running the code evalMPII.m. I have made the changes for weights. 

I tried training with complete MPII images for 4000 iterations but still the predictions are nothing. Can anyone please suggest what is going wrong?
@ZheC ....can you please provide the loss curve for MPII training dataset and prototxt file required for training. 

PS: The pretrained model gives prefect results with evalMPII.m so the evaluation procedure I am following is correct. The training is not happening properly. I am using python3 and have converted the python2 codes into 3. The .prototxt file generated is correct on comparing with the pretrained one.

Thank You

**Comment by mehtadushy** [comment]
I think the issue likely comes from the heatmap and PAF GT being generated by the CPMData layer. Have you examined the GT ?
